rabbitmq-components.mk

```diff
@@ -1,12 +1,199 @@
-# Inspired by erlang.mk bootstrap Makefile.
```

Comment:
Are changes to this file intentional or have they been included by mistake? You can instruct git to ignore changes to a file by doing: `git update-index --assume-unchanged rabbitmq-components.mk`.

Reply:
this is indeed by mistake. I myself never looked at the content of the `erlang.mk` and `rabbitmq-components.mk` and assumed they have the "usual" content instead of a dynamic bootstrap Makefile.